packages/controllers/organization_form.coffee

```diff
@@ -1,4 +1,15 @@
 if Meteor.isClient
+  Template.organizationForm.onCreated ->
+    Session.set('organization', new Organization())
+
+  Template.organizationForm.helpers
+    organization: ->
+      Session.get('organization')
```

Comment:
I saw an example that used the Session like this and it makes sense to me, but if there's an easier way to effectively have the form object be reactive I'd love to see it.